Blizzard.IO.RabbitMQ.Tests/NetqRabbitConfiguratorTests.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace Blizzard.IO.RabbitMQ.Tests
+{
+    [TestFixture]
+    public class NetqRabbitConfiguratorTests
+    {
+        private RabbitConfiguration _configuration;
+        private RabbitExchange _exchange;
+        private RabbitQueue _queue;
+
+        private Mock<IBus> _netqBusMock;
+        private Mock<IAdvancedBus> _netqAdvancedBusMock;
+        private Mock<ILogger<NetqRabbitConfigurator>> _loggerMock;
+        private NetqRabbitConfigurator _netqRabbitConfigurator;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _exchange = new RabbitExchange
+            {
+                Name = "EXCHANGE",
+                Type = RabbitExchangeType.Direct,
+                Passive = false,
+                Durable = false,
+                AutoDelete = false,
+                Internal = false,
+                AlternateExchange = "2",
+                Delayed = false
+            };
+
+            _queue = new RabbitQueue
+            {
+                Name = "QUEUE",
+                Passive = false,
+                Durable = false,
+                Exclusive = false,
+                AutoDelete = false,
+                PerQueueMessageTtl = 12,
+                Expires = 3,
+                MaxPriority = 4,
+                DeadLetterExchange = "are",
+                DeadLetterRoutingKey = "really...",
+                MaxLength = 10,
+                MaxLengthBytes = 1
+            };
+
+            _configuration = new RabbitConfiguration
+            {
+                Exchanges = new List<RabbitExchange>(),
+                Queues = new List<RabbitQueue>(),
+                ExchangeToExchangeBindings = new List<RabbitBinding>(),
+                ExchangeToQueueBindings = new List<RabbitBinding>()
+            };
+
+            _loggerMock = new Mock<ILogger<NetqRabbitConfigurator>>();
+            _netqAdvancedBusMock = new Mock<IAdvancedBus>();
+            _netqBusMock = new Mock<IBus>();
+            _netqBusMock.SetupGet(bus => bus.Advanced).Returns(_netqAdvancedBusMock.Object);
+            _netqRabbitConfigurator = new NetqRabbitConfigurator(_netqBusMock.Object, _loggerMock.Object);
```

Comment:
Add space between mock initialization and setups and between setup to tested class initialization